packages/components/stories/Notification.js

```diff
@@ -27,32 +26,38 @@ class NotificationWrapper extends React.Component {
 					},
 				},
 			]);
-			this.setState({	counter: this.state.counter + 1 });
+			this.setState(prevState => ({ counter: prevState.counter + 1 }));
 		}, 500);
 		setTimeout(() => {
 			this.notifications = this.notifications.concat([
 				{
 					id: 'story-2',
 					type: 'error',
-					message: ['This is a feedback of your operation2', 'This is a feedback of your operation1, This is a feedback of your operation1'],
+					message: [
+						'This is a feedback of your operation2',
+						'This is a feedback of your operation1, This is a feedback of your operation1',
+					],
 					action: {
 						label: 'undo',
 						icon: 'talend-undo',
 						onClick: action('click undo'),
 					},
 				},
 			]);
-			this.setState({	counter: this.state.counter + 1 });
+			this.setState(prevState => ({ counter: prevState.counter + 1 }));
```

Comment:
i don't see the point of doing that here since what the function is doing here is really simple.